src/components/Cashout/Components/Initial.view.tsx

```diff
@@ -62,12 +62,6 @@ export const InitialCashoutView = ({
         inputDenomination === 'TOKEN' ? tokenValue : usdValue
     )
 
-    useEffect(() => {
-        if (!_tokenValue) {
-            _setTokenValue('0')
-        }
-    }, [])
-
```

Comment:
Setting 0 preprends user input with 0, which differs from the behavior from other inputs

Reply:
yeah this was a bad hack by me in a previous PR - also bad that it fell through in PR review ^^

praise: nice commenting the code on github and reason for removal